paymentsheet/src/main/java/com/stripe/android/link/theme/LinkShapes.kt

```diff
@@ -1,9 +1,12 @@
 package com.stripe.android.link.theme
 
 import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.dp
 
-internal object LinkShapes {
-    val extraSmall = RoundedCornerShape(4.dp)
-    val default = RoundedCornerShape(12.dp)
-}
+internal data class LinkShapes(
+    val extraSmall: RoundedCornerShape = RoundedCornerShape(4.dp),
+    val default: RoundedCornerShape = RoundedCornerShape(12.dp),
+    val primaryButton: RoundedCornerShape = RoundedCornerShape(12.dp),
+    val primaryButtonHeight: Dp = 56.dp,
```

Comment:
👀 Yeah, height isn't a shape, but it's shape-related and this is easier than adding a new thing.